CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.14.0] - 2023-04-13
+
+### Added
+- Two new ML features are added, background blur and replacement 2.0.
```

Comment:
`Two new ML features are added` seems redundant. Can we add more details on what background blur and replacement 2.0 are?